src/main/java/com/questhelper/requirements/ItemRequirement.java

```diff
@@ -60,6 +63,7 @@
 	protected boolean highlightInInventory;
 
 	protected final List<Integer> alternates = new ArrayList<>();
+	protected final Map<Integer, String> itemNameMap = new HashMap<>();
```

Comment:
I'm not really a fan of this way of doing it, it's quietly used just by specifying a quantity but nothing else for the initial name, which isn't ideal, and ends up with replication of ids, just 1 has more info.

I think what I'd prefer is we create a class, say 'QuestItem', which will define the ID/Name relationship, and replace alternates with that. 

We can then have a boolean which explicitely states to override the text, and uses `client.getItemDefinition(ITEM_ID).getName()` to get its name. WDYT?

Reply:
Fixed. As discussed on discord.